README.md

```diff
@@ -54,13 +54,16 @@ If you ever need help, feel free to ask around on our [Discord server](https://d
 
 2. Navigate to the root directory and install the dependencies  
    `cd AntAlmanac`  
-   `npm install`
+   `npm i -g yarn` (if this fails, try running it with admin privileges)  
+   `yarn install`
 
 3. Start the development server  
    `npm start`
```

Comment:
maybe `yarn start` just to be consistent? They all do the same thing, but might as well commit to using yarn for everything if we bring it up at the beginning

Reply:
Agreed.

Reply:
Also recommending admin privileges really rubs me the wrong way; running as admin is usually an escape hatch if you didn't set up your dev environment properly. It's the easiest work-around to get started, but you really should be using [fnm](https://github.com/Schniz/fnm) or [nvm](https://github.com/nvm-sh/nvm) to manage your node installation for the current user. But maybe it doesn't seem as beginner friendly to bring that up?